evalml/tests/component_tests/test_delayed_features_transformer.py

```diff
@@ -173,3 +226,40 @@ def test_target_delay_when_gap_is_0(gap, delayed_features_data):
         answer = answer.drop(columns=["target_delay_0"])
 
     pd.testing.assert_frame_equal(transformer.fit_transform(None, y), answer)
+
+
+@pytest.mark.parametrize('use_woodwork', [True, False])
```

Comment:
Realized we didn't have coverage for woodwork

Reply:
Woot woot 🥳